shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -573,7 +578,13 @@ - (SemanticsObject*)search:(CGPoint)point {
 // IOS 16. Overrides this method to focus the first eligiable semantics
 // object in hit test order.
 - (id)_accessibilityHitTest:(CGPoint)point withEvent:(UIEvent*)event {
-  return [self search:point];
+  id hittest = [self search:point];
+  if (![hittest isKindOfClass:[SemanticsObject class]]) {
+    // If hittest result is not a SemanticsObject (e.g. PlatformView),
+    // call the default hittest method to find the hittest result inside the "hittest".
+    return [(NSObject*)hittest _accessibilityHitTest:point withEvent:event];
```

Comment:
This is not viable, unfortunately.

If we can't implement this method without needing to make a call like this, we'll need to remove the method entirely and find a different workaround for the initial problem it was added to address.